### PR TITLE
MNIST Task 1 - Number dataset

### PR DESCRIPTION
**Pull request by EeshaanJain**
Added Convolution, MaxPool, Dropout and BatchNormalisation layers. Changed epochs from 5 to 10. Used relu activation.
Accuracy achieved : train : 99.61%, test : 99.23%.

### CODE REVIEW

**Comment by dev9318** [comment]
@EeshaanJain  Change the name of the file to TASK(no. of the task)_(your initials).

**Comment by EeshaanJain** [comment]
@dev9318 Did that
